libsolidity/analysis/TypeChecker.cpp

```diff
@@ -164,10 +164,18 @@ void TypeChecker::checkContractDuplicateFunctions(ContractDefinition const& _con
 		for (; it != functions[_contract.name()].end(); ++it)
 			ssl.append("Another declaration is here:", (*it)->location());
 
+		string msg = "More than one constructor defined.";
+		size_t occurances = ssl.infos.size();
+		if (occurances > 32)
+		{
+			ssl.infos.resize(32);
+			msg += " The list is truncated from " + boost::lexical_cast<string>(occurances) + " to 32 occurances.";
```

Comment:
Perhaps shorter: `(only listing the first 32 occurrences)`?

Reply:
Shorted it to `Truncated from X to the first 32 occurrences`

Alternative: `Only showing the first 32 occurrences out of a total X.`

I think it is useful to show the total.